static/context.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import { AsyncStorage } from 'react-native'
+import FormData from 'FormData'
+
+import api from './api'
+import { content } from './content'
+
+export const AppContext = React.createContext()
+export const AppConsumer = AppContext.Consumer
+
+export class AppProvider extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      userId: ''
+    }
+  }
+
+  async login (email, password) {
+    const url = `${content.url}${content.loginPath}`
+
+    const formData = new FormData()
+    formData.append('email', email)
+    formData.append('password', password)
+
+    const postData = {
+      body: formData
+    }
+
+    try {
+      const response = await api(url, postData).bind(this)
```

Comment:
You'll really need to explain to me some day why we need to bind the current scope to the `api` function 🤔

Reply:
That was just a test (and it's not working ^^)